threeML/utils/OGIP/response.py

```diff
@@ -587,7 +587,7 @@ def _read_matrix(self,
         f_chan_column_pos = data.columns.names.index("F_CHAN") + 1
 
         try:
-            tlmin_fchan = header["TLMIN%i" % f_chan_column_pos]
+            tlmin_fchan = int(header["TLMIN%i" % f_chan_column_pos])
```

Comment:
Thanks for this! Would you mind using an f-string expression here? We are slowly trying to convert over to that.